project_department/views/project.xml

```diff
@@ -0,0 +1,83 @@
+<odoo>
+
+     <record id="edit_project" model="ir.ui.view">
+         <field name="name">project.project.form</field>
+         <field name="model">project.project</field>
+         <field name="inherit_id" ref="project.edit_project"/>
+         <field name="type">form</field>
+         <field name="arch" type="xml">
+             <field name="user_id" position="after">
+                 <field name="department_id"/>
+             </field>
+         </field>
+     </record>
+
+    <record id="view_project_project_filter" model="ir.ui.view">
+         <field name="name">project.project.select</field>
+         <field name="model">project.project</field>
+         <field name="inherit_id" ref="project.view_project_project_filter"/>
+         <field name="type">search</field>
+         <field name="arch" type="xml">
+               <field name="partner_id" position="before">
+                  <field name="department_id"/>
+               </field>
+               <xpath expr="//filter[contains(@domain, user_id)]" position="after">
```

Comment:
What are we selecting here? I don't see this filter in the orig view: https://github.com/odoo/odoo/blob/11.0/addons/project/views/project_views.xml#L175